### ISSUE TITLE
Retest Only: CAMPD Right Menu - EPA 508 issue 

### ISSUE THREAD

**Issue by JanellC** [US-EPA-CAMD/PI-1#2283]
## Finding 
| Page/ Screen | Finding | WCAG 2.0 Standard(s) |
|---|---|---|
|  Right Menu  | The right menu contains 3 subtopics each of which has both a button and a link.   Both the buttons and links have identical names but different functions.   The buttons expand and collapse the corresponding submenus while the links lead to an external information page.   These names should be updated to clarify the function of each. | 2.4.4 |

## Context 

The tech team is working on the right menu component. The new menu component will be implemented with ticket Refactor CAMPD UI to use easey-design-system components #1572. This ticket should be retested with ticket #1572

**Comment by aprematta** [comment]
Retest complete, there are no more expandable buttons, only links. Issue is fixed
